.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ci
+on: [push, pull_request]
+env:
+  COVERAGE_OPTION: ./node_modules/.bin/nyc
+  NODE_VERSION: 10
+  PARSE_SERVER_TEST_TIMEOUT: 20000
+jobs:
+  check-mongo:
+    strategy:
+      matrix:
+        include:
+          - name: Mongo 4.0.4, ReplicaSet, WiredTiger
+            MONGODB_VERSION: 4.0.4
+            MONGODB_TOPOLOGY: replicaset
+            MONGODB_STORAGE_ENGINE: wiredTiger
+            NODE_VERSION: 10
+          - name: Mongo 3.6.9
+            MONGODB_VERSION: 3.6.9
+            NODE_VERSION: 10
+          - name: Redis Cache
+            PARSE_SERVER_TEST_CACHE: redis
+            NODE_VERSION: 10
+          - name: Node 12.12.0
+            NODE_VERSION: 12.12.0
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-16.04
```

Comment:
Based also on LTS version and support max dates may be we should use 
ubuntu 20.04

Reply:
done